README.md

```diff
@@ -20,22 +20,25 @@ This image is published to Docker Hub and can be pulled via the command
 docker pull ppiper/node-browsers
 ```
 
-The default tag contains node 10 due to compatibility reasons with approuter.
+The default tag contains node 10 due to compatibility reasons with approuter, it is EOL: 2021-04-01
 
-For node 12, use the tag `node12`:
+For node 12 or node 14, use the tag `12-buster` or `14-buster`:
 
 ```
-docker pull ppiper/node-browsers:node12
+docker pull ppiper/node-browsers:12-buster
+docker pull ppiper/node-browsers:14-buster
 ```
 
 ## Build
 
 To build this image locally, open a terminal in the directory of the Dockerfile and run
 
 ```
-docker build -t ppiper/node-browsers .
+docker build --build-arg=BASE_IMAGE=node:10-buster -t ppiper/node-browsers .
```

Comment:
This looks strange. Maybe without the "=node:10-buster"?

Reply:
we need this, this is the whole trick why we can build from multiple base images with a single dockerfile